spec/jobs/nb_person_push_job_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe NbPersonPushJob, type: :job do
+  describe "#perform" do
+    it "calls NationBuilder with correct args" do
+      args = { attributes: { email: "user@example.com", phone: '510-555-9999' } }
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.